eclass/kernel-build.eclass

```diff
@@ -436,6 +436,36 @@ kernel-build_merge_configs() {
 		fi
 	fi
 
+	if [[ ${KERNEL_IUSE_SECUREBOOT} ]]; then
+		if use secureboot; then
+			# This only effects arm64 and riscv where the bootable image may
+			# contain its own decompressor (zboot). If enabled, both the
+			# decompressor and the embedded kernel must be signed, the kernel
+			# build system can take care of this.
+			cat <<-EOF > "${WORKDIR}/secureboot.config" || die
+				## Enable zboot signing
```

Comment:
No reason. We can also change the default over at https://github.com/projg2/gentoo-kernel-config later. However, we might want to be conservative and leave it conditional to `USE=secureboot` because I am not 100% sure all arm64/riscv boards implement UEFI and will be able to load the `vmlinuz.efi`. If however we are on a system where USE=secureboot is supported then it must be UEFI capable so then we can be sure it will work.